store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStorePublicAccessTest.java

```diff
@@ -53,20 +53,22 @@
     public void setUp() throws Exception {
         CacheServiceLoader.shutdown();
         fedProps = new FederatedStoreProperties();
-        fedProps.setCacheProperties(CACHE_SERVICE_CLASS_STRING);
+        fedProps.setCacheServiceClass(CACHE_SERVICE_CLASS_STRING);
 
         store = new FederatedStore();
         library = new HashMapGraphLibrary();
         HashMapGraphLibrary.clear();
 
         library.addProperties(PROP_1, PROPERTIES);
         library.addSchema(SCHEMA_1, new Schema.Builder().build());
-        store.setGraphLibrary(library);
```

Comment:
I presume this can no longer be done before each?

Reply:
Been so long can't remember will need to investigate